x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -68,3 +84,72 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
     </EuiFormRow>
   );
 }
+
+export function XyToolbar(props: VisualizationToolbarProps<State>) {
+  const [open, setOpen] = useState(false);
+  const hasNonBarSeries = props.state?.layers.some(
+    (layer) => layer.seriesType === 'line' || layer.seriesType === 'area'
+  );
+  return (
+    <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlexItem grow={false}>
```

Comment:
Unnecessary flex group with only a single flex item.

Reply:
It's there to align it with the right boundary of the preview (`flexEnd` in the line above). Also I expect this to have more items soon. Should it be solved in another way?

Reply:
Ah you are correct. 👍